package/batocera/utils/gpicase/batocera-gpicase-install

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 CONFIGFILE=/boot/config.txt
-CONFIG="$(batocera-settings -f "$CONFIGFILE" -r force_gpi_defaults)"
+CONFIG="$(/usr/bin/batocera-settings-get -f "$CONFIGFILE" force_gpi_defaults)"
 CONFIGERR=$?
 [[ $CONFIGERR -eq  12 || $CONFIG -eq 1 ]] || exit 0
```

Comment:
Pay attention to the error output.
There is later a switch that checks if the entry is inside the /boot/config.boot

Can you later write which errorcodes are returned from your script? And what they mean?

Reply:
The new `batocera-settings-get` returns `1` if the key or config file is missing or unreadable
If the config file a missing or unreadable, an error is also printed to stderr.

Fixed the `batocera-gpicase-install` script in [`02c53b2` (#2681)](https://github.com/batocera-linux/batocera.linux/pull/2681/commits/02c53b2d643529901ee27cc001a1405498e2da57)